mopro-ffi/SwiftBindings/mopro.swift

```diff
@@ -21,21 +21,21 @@ private extension RustBuffer {
     }
 
     static func empty() -> RustBuffer {
-        RustBuffer(capacity: 0, len: 0, data: nil)
+        RustBuffer(capacity: 0, len:0, data: nil)
     }
 
     static func from(_ ptr: UnsafeBufferPointer<UInt8>) -> RustBuffer {
-        try! rustCall { ffi_mopro_bindings_rustbuffer_from_bytes(ForeignBytes(bufferPointer: ptr), $0) }
+        try! rustCall { ffi_mopro_ffi_rustbuffer_from_bytes(ForeignBytes(bufferPointer: ptr), $0) }
```

Comment:
please revert this file change 🙏🏻 (and also  `moproFFI.h`)
It would be confusing when using `./build_bindings.sh`
If there is no ffi changes, please just keep the previous one

> [!NOTE]  
> We need to change the `mopro-ffi/Cargo.toml` to
> ```toml
> [lib]
> name = "mopro_bindings"
> ```
> And the functions would be `ffi_mopro_bindings_{}`
> but later on, the `test-e2e` uses `mopro_ffi` lib name
> so it has to switch to `name = "mopro_ffi"`
> 
> The solution could be
> 1. just using `mopro_ffi` as the lib name
> 2. config lib name in [`uniffi.toml`](https://mozilla.github.io/uniffi-rs/latest/swift/configuration.html)

Reply:
Reverted